posts/2024-02-15-Windows-7.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "New Windows 7 targets and tier 1 deprecation"
+author: Chris Denton
+---
+
+The minimum requirements for tier 1 toolchains targeting Windows will increase with the 1.77 release (scheduled for March 21, 2024).
+Windows 10 will now be the minimum supported version.
+These requirements apply both to running the Rust compiler itself and to running binaries produced by Rust, if they use the libstd.
+
+Additionally there are two new targets that have Windows 7 as their baseline: `x86_64-win7-windows-msvc` and `i686-win7-windows-msvc`.
```

Comment:
Maybe list which tier they are?

Reply:
it would be worth explicitly mentioning what tier 3 means here (not guaranteed to build, **no build artifacts provided**) and mention whether there are plans to upgrade them to tier 2, providing artifacts for users

Reply:
I'd love them to be tier 2 but I think the compiler team would need to weigh on that one?

Reply:
Ok, I've ambiguously said they're "starting" as tier 3. Which implies they might be promoted but doesn't actually say that. I can update it to be less squirely if/when there's more information.